macros/AddonSidebar.ejs

```diff
@@ -355,8 +356,9 @@ var text = mdn.localStringMap({
     </li>
     <li class="toggle">
       <details>
-        <summary><%=text['WebExtensions#Firefox_workflow']%></summary>
+        <summary><a href="<%=baseURL%>WebExtensions#Firefox_workflow"><%=text['WebExtensions#Firefox_workflow']%></a></summary>
```

Comment:
As also commented in https://github.com/mdn/kumascript/pull/921: I don't think we should be doing this.

Reply:
Sorry, removed